torch/ao/nn/sparse/quantized/utils.py

```diff
@@ -36,10 +38,15 @@ def __init__(self, row_block_size=1, col_block_size=4):
         LinearBlockSparsePattern.row_block_size = row_block_size
         LinearBlockSparsePattern.col_block_size = col_block_size
 
-    def __enter__(self):
+    def __enter__(self) -> None:
         pass
 
-    def __exit__(self, exc_type, exc_value, backtrace):
+    def __exit__(
+        self,
+        exc_type: Optional[Type[BaseException]],
```

Comment:
FYI: Torch jit previously throw a fit with proper `__exit__` typing, not sure if that's been fixed.